pkg/frontend/load.go

```diff
@@ -1373,6 +1385,7 @@ func writeBatchToStorage(handler *WriteBatchHandler, force bool) error {
 			}
 		}
 	}
+	handler.threadCnt = 0
```

Comment:
1. Update the shared variable with sync.atomic  in the concurrent environment.
2. Put the access to the threadCnt just after the tableHandler.Write.